frontends/web/src/utils/transport-android.ts

```diff
@@ -15,20 +15,20 @@
  * limitations under the License.
  */
 
-import { TMsgCallback, TQueryPromiseMap } from './transport-common';
+import { TMsgCallback, TPayload, TQueryPromiseMap } from './transport-common';
 import { runningInAndroid } from './env';
 
 let queryID: number = 0;
 const queryPromises: TQueryPromiseMap = {};
-const currentListeners: Function[] = [];
+const currentListeners: TMsgCallback[] = [];
 
-export function androidCall(query: string) {
+export function androidCall(query: string): Promise<any> {
```

Comment:
Before TS somehow automatically used `Promise<unknown>` which is in theory stricter than `Promise<any>`.

![Screen Shot 2023-12-24 at 09 37 00](https://github.com/digitalbitbox/bitbox-wallet-app/assets/546900/460bd0c3-e9de-4461-bec2-164cfc96e86d)

How about using `Promise<unknown>` here? Although it doesn't matter much I think as we set the correct return type in api/ files. I did a quick test and weblint didn't complain using `Promise<unknown>`.  I slightly prefer unkown here.

Reply:
Sure. It also matches the type in `TQueryPromiseMap`

https://github.com/digitalbitbox/bitbox-wallet-app/blob/master/frontends/web/src/utils/transport-common.ts#L53